backend/Squirrel.Core/Squirrel.Core.Common/Enums/EngineEnum.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Squirrel.Core.Common.Enums
+{
+    public enum EngineEnum
+    {
+        SqlServer,
+        Postgre
```

Comment:
probably `PostgreSQL` if you mean `https://www.postgresql.org` ?

Reply:
> probably `PostgreSQL` if you mean `https://www.postgresql.org` ?

Yes, I'll fix it now